.github/workflows/ci.yaml

```diff
@@ -68,13 +68,13 @@ jobs:
   release:
     name: release
     needs: [build]
+    outputs:
+      container_digest: ${{ steps.container_info.outputs.container_digest }}
+      container_tags: ${{ steps.container_info.outputs.container_tags }}
+      container_repos: ${{ steps.container_info.outputs.container_repos }}
 
     runs-on: ubuntu-20.04
 
-    permissions:
```

Comment:
I prefer to keep the explicit permissions like we do in other projects as well to adopt these practices.

Reply:
I agree!

Reply:
Merged with main which adds fine grained perms